src/book/entities/Account.ts

```diff
@@ -116,7 +116,6 @@ export default class Account extends BaseEntity {
     nullable: true,
   })
   @IsOptional()
-  @Length(4, 2048)
```

Comment:
This length validators I just found them to be annoying and don't bring much value so removing